```
        # Check the Dataset has been duplicated
>       assert len(objsd) == 2
E       assert 1 == 2
E         +1
E         -2
```
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/lastCompletedBuild/testReport/OmeroPy.test.integration.clitest.test_duplicate/TestDuplicate/testSkipheadDuplication/

--- a/components/tools/OmeroJava/test/integration/DuplicationTest.java
+++ b/components/tools/OmeroJava/test/integration/DuplicationTest.java
@@ -16,6 +16,7 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
+
 package integration;
 
 import java.util.ArrayList;
@@ -718,7 +719,7 @@ public class DuplicationTest extends AbstractServerTest {
 
 
     /**
-     * Test duplication of an annotated image such that it does not duplicate an attachment.
+     * Test duplication of an image along with its attachment.
      * @throws Exception unexpected
      */
     @Test

--- a/components/tools/OmeroJava/test/integration/DuplicationTest.java
+++ b/components/tools/OmeroJava/test/integration/DuplicationTest.java
@@ -1242,7 +1242,7 @@ public class DuplicationTest extends AbstractServerTest {
      * Test duplication of a dataset's image. {@link SkipHead} is used to identify the image via its dataset.
      * @throws Exception unexpected
      */
-    @Test(groups = {"ticket:13197", "broken"})
+    @Test(groups = "ticket:13197")
     public void testDuplicateImageViaSkipHead() throws Exception {
         newUserAndGroup("rwr---");
 

